internal/api/core/kubernetes/delete.go

```diff
@@ -31,20 +31,21 @@ func (cc *Controller) Delete(c *gin.Context, dm DeleteManifestRequest) {
 	}
 
 	do := metav1.DeleteOptions{}
-
 	if dm.Options.GracePeriodSeconds != nil {
 		do.GracePeriodSeconds = dm.Options.GracePeriodSeconds
 	}
-	// There are two options that will set the deletion to cascade. Originally this was set when
-	// `options.cascading = true`, however in later APIs they have this set in
-	// `options.orphanDependants = false`.
+	// There are two options that will set the deletion to orphan dependants:
+	// `options.cascading = false` and `options.orphanDependants = true`. The first
+	// happens when running a Delete (Manifest) stage and the second
```

Comment:
Change from:
happens when running a Delete (Manifest) stage
to:
happens when running a Delete (Manifest) stage runs or executes